Makefile

```
@@ -21,14 +21,16 @@ install:
 		$(DESTDIR)/usr/lib/systemd/system-generators \
 		$(DESTDIR)/usr/lib/tmpfiles.d \
 		$(DESTDIR)/etc/env.d \
-		$(DESTDIR)/usr/share/ssh
+		$(DESTDIR)/usr/share/ssh \
+		$(DESTDIR)/usr/lib/modules-load.d
 	install -m 755 bin/* $(DESTDIR)/usr/bin
 	install -m 755 sbin/* $(DESTDIR)/usr/sbin
 	ln -sf flatcar-install $(DESTDIR)/usr/bin/coreos-install
 	install -m 755 scripts/* $(DESTDIR)/usr/lib/flatcar
 	install -m 644 udev/rules.d/* $(DESTDIR)/lib/udev/rules.d
 	install -m 755 udev/bin/* $(DESTDIR)/lib/udev
 	install -m 644 configs/editor.sh $(DESTDIR)/etc/env.d/99editor
+	install -m 644 configs/k8s.conf $(DESTDIR)/usr/lib/modules-load.d/
```

```suggestion
	install -m 644 configs/modules-load.d/* $(DESTDIR)/usr/lib/modules-load.d/
```
We can use a subfolder for future additions.

---

Excellent suggestion yes.